.github/workflows/test.yml

```diff
@@ -0,0 +1,143 @@
+name: test
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'ghactions-test' # remove when merged
+    paths-ignore:
+      - '**.md'
+      - '.github/workflows/doc.yml'
+      - 'doc/**'
+      - 'mkdocs.yml'
+  pull_request:
+    branches:
+      - 'master'
+    paths-ignore:
+      - '**.md'
+      - '.github/workflows/doc.yml'
+      - 'doc/**'
+      - 'mkdocs.yml'
+
+env:
+  APP_ENV: testing
+  DB_NAME: db
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
We want to be able to use different version here. 
Check https://github.com/librenms/librenms/pull/12035

Tldr is that we want to test different net-snmp and mysql versions

Reply:
If you just want to test other releases of mysql you just have to change the version of the mysql Docker service (or create a new one with your desired release). This way we are sure of the current MySQL release we want to use: https://github.com/librenms/librenms/pull/12392/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R46

Reply:
I agree with SQL versions supplied via the services tag.

However net-snmp version changes shouldnt really matter much although we should keep track of the LTS versions of ubuntu just to be sure that compatibility is ensured.

Reply:
I'm fine with using a new version, just aware that some test data will fail.